### ISSUE TITLE
Add to the UI a way to filter assets based on asset's status

### ISSUE THREAD

**Issue by Calvo94** [dagster-io/dagster#14943]
### What's the use case?

Hi team, one feature that can be helpful for our team is to add to the UI a way to filter assets based on asset's status. I want to be able, from the UI, to answer the following questions :
Based on changes since last materialization:
Which assets have a new code version?
Which assets have their upstream assets have a new code version?
Which assets have a new data version?
Which assets have their upstream assets have a new data version?

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by nixent** [comment]
@smackesey @bengotow it would be nice to have simple filter on failed assets, similar to one available for Runs

**Comment by cyberjar09** [comment]
yes would be very useful indeed to be able to filter assets based on the status
